pstress/generate_mysqld_options.sh

```diff
@@ -0,0 +1,511 @@
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-log-files-in-group" ]; then
+    echoit "  > Adding possible values 0,1,2,5,10 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=0" >> ${OUTPUT_FILE}
+    echo "${OPTION}=1" >> ${OUTPUT_FILE}
+    echo "${OPTION}=2" >> ${OUTPUT_FILE}
+    echo "${OPTION}=5" >> ${OUTPUT_FILE}
+    echo "${OPTION}=10" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-checksum-algorithm" ]; then
+    echoit "  > Adding possible values innodb, crc32 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=innodb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=crc32" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-cleaner-lsn-age-factor" ]; then
+    echoit "  > Adding possible values legacy, high_checkpoint for option '${OPTION}' to the final list..."
+    echo "${OPTION}=legacy" >> ${OUTPUT_FILE}
+    echo "${OPTION}=high_checkpoint" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-corrupt-table-action" ]; then
+    echoit "  > Adding possible values assert, warn, salvage for option '${OPTION}' to the final list..."
+    echo "${OPTION}=assert" >> ${OUTPUT_FILE}
+    echo "${OPTION}=warn" >> ${OUTPUT_FILE}
+    echo "${OPTION}=salvage" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-empty-free-list-algorithm" ]; then
+    echoit "  > Adding possible values legacy, backoff for option '${OPTION}' to the final list..."
+    echo "${OPTION}=legacy" >> ${OUTPUT_FILE}
+    echo "${OPTION}=backoff" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-buffer-pool-evict" ]; then
+    echoit "  > Adding possible value uncompressed for option '${OPTION}' to the final list..."
+    echo "${OPTION}=uncompressed" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-flush-method" ]; then
+    echoit "  > Adding possible values fsync, O_DSYNC for option '${OPTION}' to the final list..."
+    echo "${OPTION}=fsync" >> ${OUTPUT_FILE}
+    echo "${OPTION}=O_DSYNC" >> ${OUTPUT_FILE}
+    echo "${OPTION}=O_DIRECT" >> ${OUTPUT_FILE}
+    echo "${OPTION}=O_DIRECT_NO_FSYNC" >> ${OUTPUT_FILE}
+    echo "${OPTION}=littlesync" >> ${OUTPUT_FILE}
+    echo "${OPTION}=nosync" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-checksum-algorithm" ]; then
+    echoit "  > Adding possible values none, innodb, crc32 for option '${OPTION}' to the final list..."
+    echo "${OPTION}=innodb" >> ${OUTPUT_FILE}
+    echo "${OPTION}=crc32" >> ${OUTPUT_FILE}
+    echo "${OPTION}=none" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-disable" ]; then
+    echoit "  > Adding possible values counter, module, pattern and all for option '${OPTION}' to the final list..."
+    echo "${OPTION}=counter" >> ${OUTPUT_FILE}
+    echo "${OPTION}=module" >> ${OUTPUT_FILE}
+    echo "${OPTION}=pattern" >> ${OUTPUT_FILE}
+    echo "${OPTION}=all" >> ${OUTPUT_FILE}
+  elif [ "${OPTION}" == "--innodb-monitor-enable" ]; then
```

Comment:
Instead of 3 seperate elif block for --innodb-monitor-enable, --innodb-monitor-reset, --innodb-monitor-reset-all, you can just have one elif block
eg.
`elif [ "${OPTION}" == "--innodb-monitor-enable" -o "${OPTION}" == "--innodb-monitor-reset" -o "${OPTION}" == "--innodb-monitor-reset-all" ]; then`

Please test above before adding

Reply:
code tested and it is working as expected